components/react/authSignerModal.tsx

```
@@ -238,39 +242,78 @@
-    if (visible) {
-      modal.showModal();
-    } else {
-      modal.close();
-    }
-  }, [visible]);
-
+  const walletIcon = wallet?.logo;
   const walletIconString = walletIcon?.toString() ?? '';
 
+  function handleReject() {
+    reject && reject();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)